packages/react/components/skeleton/src/Skeleton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { cx } from '@project44-manifest/react-styles';
+import type { ForwardRefComponent } from '@project44-manifest/react-types';
+import { StyledSkeleton } from './Skeleton.styles';
+import type { SkeletonElement, SkeletonProps } from './Skeleton.types';
+
+export const Skeleton = React.forwardRef((props, forwardedRef) => {
+  const { children, className: classNameProp, css, ...other } = props;
+
+  const id = React.useId();
+
+  const className = cx('manifest-skeleton', classNameProp);
+
+  return (
+    <StyledSkeleton
+      {...other}
+      className={className}
+      css={css}
+      uniqueKey={id} // needed for SSR: https://github.com/danilowoz/react-content-loader#server-side-rendering-ssr---match-snapshot
+    >
+      {children}
+    </StyledSkeleton>
```

Comment:
I could not satisfy Typescript until removing `ref` and `as`. Looking for suggestions here on the best way to proceed

Reply:
I image it having to do with the `ContentLoader` component yeah? Maybe we don't want to support polymorphism with this component since its a third-party wrapper.

Reply:
Agree, let me get that updated and pull out the forwardRef types

Reply:
@branclar updated; refactored into normal react component without forwardRef